.travis.yml

```diff
@@ -44,10 +44,9 @@ env:
 
   matrix:
     # what we used for the latest PMM release
-    - GO_VERSION=1.13.x
-
-    # the latest released version
     - GO_VERSION=1.14.x
+    # the latest HEAD
+    - GO_VERSION=tip TEST_GROUP=canFail
```

Comment:
https://github.com/percona/pmm-agent/blob/6efdebc931d664ba058401194afa613ff648b1ce/.travis.yml#L12-L15

Without `TEST_GROUP`, and with tip instead of master.

Please also add a blank line before that group

Reply:
err, it should be something like
```
jobs:
  allow_failures:
    - env: GO_VERSION=tip
```